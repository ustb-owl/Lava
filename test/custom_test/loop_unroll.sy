void test(int n) {
    int i, j;
    
    i = 0;
    j = 0;
    while (i < n) {
        j = j + i;
        i = i + 1;
    }
    putint(j);
    putch(10);
}

int main() {
    test(-1);
    test(0);
    test(1);
    test(2);
    test(100);
    test(101);
    return 0;
}