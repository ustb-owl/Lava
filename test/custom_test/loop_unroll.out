0
0
0
1
4950
5050
0
